src/backend/distributed/executor/multi_utility.c

```diff
@@ -188,6 +188,14 @@ multi_ProcessUtility(Node *parsetree,
 		return;
 	}
 
+	if (!CitusHasBeenLoaded())
+	{
+		standard_ProcessUtility(parsetree, queryString, context,
```

Comment:
It's not obvious that creating the Citus extension changes PostgreSQL's behaviour even if you have no distributed tables, so this might be the right place to communicate that. Could use better wording.